oauth-common/src/main/java/io/strimzi/kafka/oauth/common/Config.java

```diff
@@ -23,6 +23,8 @@
     public static final String OAUTH_SSL_SECURE_RANDOM_IMPLEMENTATION = "oauth.ssl.secure.random.implementation";
     public static final String OAUTH_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM = "oauth.ssl.endpoint.identification.algorithm";
     public static final String OAUTH_ACCESS_TOKEN_IS_JWT = "oauth.access.token.is.jwt";
+    public static final String OAUTH_CONNECT_TIMEOUT = "oauth.connect.timeout";
+    public static final String OAUTH_READ_TIMEOUT = "oauth.read.timeout";
```

Comment:
Would it make sense to add the unit into the option name? E.g. `oauth.connect.timeout.seconds` and `oauth.read.timeout.seconds`? It makes it more readable and both Kafka and Strimzi seem to use this in many places.

Reply:
Actually it makes a lot of sense since there are already `oauth.jwks.refresh.seconds`, `oauth.jwks.expiry.seconds`, and `oauth.jwks.refresh.min.pause.seconds`.